org_status/org_status.py

```diff
@@ -75,10 +76,24 @@ def get_argument_parser():
     parser.add_argument('--verbose', '-v', action='store_true')
     parser.add_argument('--hosts-only', '-o', action='store_true')
     parser.add_argument('--skip-host-checks', action='store_true')
+    parser.add_argument('--exporter', type=str)
```

Comment:
Need a better options structure. Since the option you are adding specifically exports org list maybe have a  `--export-orgs` option taking file name and `--format` option to actually select the format of encoding?

Reply:
People shouldn't be able to choose the format of the output as the tool should decide that. We wouldn't want somebody exporting json when the tool (like gitman) is expecting a yaml file.

Reply:
I assume that by format of encoding you mean yaml/json?